packages/nx-sonarqube/src/executors/scan/schema.d.ts

```diff
@@ -11,4 +11,6 @@ export interface ScanExecutorSchema {
   skipImplicitDeps?: boolean;
   testInclusions?: string;
   verbose?: boolean;
+  excludedOptions?: string[];
```

Comment:
why is `excludedOptions` needed?

Reply:
This is an artifact when I first thought of an implementation. It is no longer needed. I will remove it.

Reply:
This is an artifact from an early implementation. It is no longer needed. It will be removed